webapp/templates/layout.html

```diff
@@ -10,8 +10,6 @@
         <link rel="shortcut icon" href="static/raspberry-pi-robot.ico" />
     </head>
     <body class="has-background-black-ter">
-        <script src="/static/scripts/hamburger.js"></script>
```

Comment:
why did you take out the hamburger button/script? I was going to replace Bulma's awkward nav menu with the sidebar.
Note: Bulma's hamburger button mandated to live on the right side of the nav bar, while the side bar that the hamburger button opens up is stashed/animated from the left (like literally every app I know)

Reply:
we can force Bulma not to generate their hamburger button in favor for ours

Reply:
I was going to integrate the new nav bar first, and then add the hamburger back once the upgrade is mostly complete

Reply:
I found a way for Bulma's QuickView to be left-justified! It's not documented but it does exist as a CSS class called `is-left`.

Link: https://github.com/Wikiki/bulma-quickview/blob/master/src/sass/index.sass#L38.